### PR TITLE
ACFT HF NLP Component release 0.0.30

### PR DESCRIPTION
**Pull request by skanakamedal**
This includes support of Flash Attention v2 to ACFT NLP components.

### CODE REVIEW

File: assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py [hunk 1]
```diff
@@ -198,6 +199,11 @@
     HfModelTypes.FALCON,
 ]
 
+FORCE_FLASH_ATTENTION_2_MODEL_TYPES = [
```
Comment (jpmann) [inline assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py:202]: add mistral model here

File: assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py [hunk 2]
```diff
@@ -63,9 +61,11 @@
 ]
 PINNED_PACAKGES = ["transformers==4.34.0", "mlflow==2.8.0"]
```
Comment (jpmann) [inline assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py:62]: removing version pinning for mlflow

File: assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py (hunk 1 above)
Comment (jpmann) [reply assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py:202]: add mistral for gc as well

Comment (skanakamedal) [reply assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py:200]: done

File: assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py (hunk 2 above)
Comment (skanakamedal) [reply assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py]: done

**Review by Anubha98 — APPROVED** [review]

**Review by jpmann — APPROVED** [review]

**Pull request merged**